app/components/structure/contact_form_modal_component.html.erb

```diff
@@ -3,7 +3,7 @@
     <h1><%= contact_sdr %></h1>
   <% end %>
   <% component.with_body do %>
-    <%= tag.turbo_frame id: 'contact-form', src: new_contacts_path do %>
+    <%= tag.turbo_frame id: 'contact-form', src: new_contacts_path, loading: 'lazy' do %>
```

Comment:
This should be lazy so only loaded when needed. This change was made to avoid flash getting rendered here.